src/lib/Input/_story.js

```diff
@@ -46,11 +82,89 @@ storiesOf('Input', module)
     type='password'
   ></Input>
 ))
-
+.add('password with placeholder', () => (
+  <Input
+    type='password'
+    placeholder={`Example Placeholder`}
+  ></Input>
+))
+.add('password (required)', () => (
+  <Input
+    type='password'
+    required
+  ></Input>
+))
+.add('password (Validation) Must Use One CAPITAL letter', () => (
```

Comment:
@Mookh01 Are we adding this to this PR? Is this part of the EIN stuff or seperate?